### ISSUE TITLE
Responsiveness of various webpages

### ISSUE THREAD

**Issue by bhutani2002** [CircuitVerse/CircuitVerse#2918]
**Describe the bug**
Various Webpages are not responsive beyond certain dimensions or completely.

"Not Responsive below 500 dimension on Width."

Dash Board:
![CircuitVerse-Dashboard-1-Old (1)](https://user-images.githubusercontent.com/84590758/153445808-9473375b-60bb-4155-a979-e86a9982afc7.png)
![CircuitVerse-Dashboard-2-Old (1)](https://user-images.githubusercontent.com/84590758/153445836-76c0d052-5fea-4a0a-bace-9f50851fd4e7.png)

"Not Responsive below 350 dimension on Width."

Home Page:
![CircuitVerse-HomePage-Old (3)](https://user-images.githubusercontent.com/84590758/153439032-c0cbf710-e462-43f9-875e-a98892721ccf.png)

Examples Page:
![CircuitVerse-ExamplesPage-Old (1)](https://user-images.githubusercontent.com/84590758/153445459-d34886ea-118d-4134-bc06-8863896d3147.png)

Editor Page:
![CircuitVerse-EditorsPage-Old (1)](https://user-images.githubusercontent.com/84590758/153445491-eeb3f93e-e632-4796-a348-6ad7618cdc43.png)

Contributor Page:
![CircuitVerse-ContributePage-Old (1)](https://user-images.githubusercontent.com/84590758/153445560-c944d2b8-734e-43a8-8195-ac606d780e78.png)

About Page:
![CircuitVerse-AboutUsPage-Old (1)](https://user-images.githubusercontent.com/84590758/153445626-c49197c6-a1da-443f-b462-072201d54987.png)

Login Page:
![CircuitVerse-LoginPage-Old (1)](https://user-images.githubusercontent.com/84590758/153445661-f4af814b-d4f9-430f-b00a-4a26f6b4ae4c.png)

SignUp Page:
![CircuitVerse-SignUpPage-Old (1)](https://user-images.githubusercontent.com/84590758/153445680-8186d7bf-64d5-4705-b039-47d340a588a6.png)

SearchBars:
![CircuitVerse-SearchBar-1-Old (1)](https://user-images.githubusercontent.com/84590758/153445729-93854741-2ed3-4279-a564-68c2f29b3d54.png)
![CircuitVerse-SearchBar-2-Old (1)](https://user-images.githubusercontent.com/84590758/153445756-a8168df0-4a61-4c96-a4fa-590de50095d9.png)

**To Reproduce**
1.(i) Go to [https://circuitverse.org/](url)
  (ii) Go to [https://circuitverse.org/examples](url)
  (iii) Go to [https://circuitverse.org/featured_circuits](url)
  (iv) Go to [https://circuitverse.org/contribute](url)
  (v) Go to [https://circuitverse.org/about](url)
  (vi) Go to [http://localhost:3000/users/sign_in](url)
  (vii) Go to [http://localhost:3000/users/sign_up](url)
  (viii) Go to [https://circuitverse.org/users/120441](url)
2. Check Responsive behaviour.
3. See error

**Expected behavior**
All the pages should be made completely responsive.

**Are you working on this issue?**
Yes

**Comment by bhutani2002** [comment]
Hey @gr455  @vedant-jain03 , please try to review it once and provide me a feedback so that I can complete the task and generate a pull request.

**Comment by vedant-jain03** [comment]
@gr455 I think that this need to be done. WDYT?

**Comment by gr455** [comment]
> @gr455 I think that this need to be done. WDYT?

Yep for sure

**Comment by kushalShukla-web** [comment]
is anyone working on this , or can i

**Comment by bhutani2002** [comment]
> is anyone working on this , or can i

Yes @kushalShukla-web I have been working on it. And the pull request is already been generated and is on the edge of merging.
Thank You.
